com.unity.render-pipelines.core/Editor/Common/Settings/SerializedScalableSettingValue.cs

```diff
@@ -1,14 +1,14 @@
 using System;
 using UnityEngine;
 using UnityEngine.Assertions;
-using UnityEngine.Rendering.HighDefinition;
+using UnityEngine.Rendering;
 
-namespace UnityEditor.Rendering.HighDefinition
+namespace UnityEditor.Rendering
 {
     /// <summary>
     /// Serialized version of <see cref="ScalableSettingValue{T}"/>.
     /// </summary>
-    internal class SerializedScalableSettingValue
+    public class SerializedScalableSettingValue
```

Comment:
Some method need documentation now that it is public but it is in collapsed area and I cannot pin every of them